src/tools/run-make-support/src/external_deps/cygpath.rs

```diff
@@ -22,14 +20,17 @@ use crate::util::handle_failed_output;
 #[track_caller]
 #[must_use]
 pub fn get_windows_path<P: AsRef<Path>>(path: P) -> String {
-    let caller = panic::Location::caller();
-    let mut cygpath = Command::new("cygpath");
-    cygpath.arg("-w");
-    cygpath.arg(path.as_ref());
-    let output = cygpath.run();
-    if !output.status().success() {
-        handle_failed_output(&cygpath, output, caller.line());
+    // If the path looks unixy then use cygpath otherwise return it unchanged.
+    // If cygpath fails then fallback to just using the path as given.
+    if path.as_ref().starts_with("/") {
```

Comment:
Discussion: actually I'm not super sure if we actually need `cygpath -w`, IIRC they are only used when interfacing with maybe msvc I think? Maybe it's worth running a try job to see if we even need the `get_windows_path` dance at all.

Reply:
Ok, as a test I'll make `get_windows_path` just return that path as-is. If that works then I'll remove it entirely.